<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AfterCompanyLogin</class>
 <widget class="QMainWindow" name="AfterCompanyLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
	background-color: qlineargradient(spread:pad, x1:0.280788, y1:0.0113636, x2:0.330039, y2:1, stop:0 rgba(161, 228, 231, 255), stop:1 rgba(255, 255, 255, 255));
}

.QLineEdit{
border-radius: 10px ;
}

.QPushButton{
border-radius: 10px ; 
background-color:rgb(124, 255, 222) ; 
}</string>
   </property>
   <widget class="QPushButton" name="searchIcon">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>35</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:3px;
background-image: url(:/images/icons8-search-30.png);
background-repeat:no-repeat;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-repeat:no-repeat;
background-image: url(:/images/travelSphere.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>771</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:grey;
font-family:times new roman;
border-radius:30px;

background-color: rgb(251, 255, 225);</string>
    </property>
   </widget>
   <widget class="QLabel" name="list_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
font-family:times new roman</string>
    </property>
    <property name="text">
     <string>LIST OF COSTUMERS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="customers">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>View Customers</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit The Table</string>
    </property>
   </widget>
   <widget class="QLabel" name="companyUsername">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#companyUsername{
font-family: 'Georgia';
font-style: normal;
font-weight: 700;
font-size: 29.8444px;
line-height: 34px;
/* identical to box height */

display: flex;
align-items: center;
text-align: center;
color: #002D62 ; 
background: linear-gradient(180deg, #258AE7 0%, rgba(175, 204, 232, 0.95) 100%);
-webkit-background-clip: text;
-webkit-text-fill-color: transparent;
background-clip: text;
text-fill-color: transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="WelcomeNote">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#WelcomeNote{
font-family: 'Georgia';
font-style: normal;
font-weight: 700;
font-size: 29.8444px;
line-height: 34px;
/* identical to box height */

display: flex;
align-items: center;
text-align: center;
color: #002D62 ; 
background: linear-gradient(180deg, #258AE7 0%, rgba(175, 204, 232, 0.95) 100%);
-webkit-background-clip: text;
-webkit-text-fill-color: transparent;
background-clip: text;
text-fill-color: transparent;
}</string>
    </property>
    <property name="text">
     <string>WELCOME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signout_icon">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>600</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(C:/Users/PC/OneDrive/Desktop/ENGT presentation/signoutvector.png) ;
background-repeat: norepeat ;
background-color: none  ;
border-radius : 5px ; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="signout">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>600</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Sign Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="icon_customerUserName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CustomerUserName">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_order">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_LineEdit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>231</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(150, 150, 255,50);
color:grey;
border-radius:10px;
font-family:Andale Mono;</string>
    </property>
    <property name="placeholderText">
     <string>Search in Database</string>
    </property>
   </widget>
   <widget class="QLabel" name="email_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>660</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font-family: 'Georgia';
font-style: normal;
font-weight: 700;
font-size: 29.8444px;
line-height: 34px;
/* identical to box height */

display: flex;
align-items: center;
text-align: center;
color: #002D62 ; 
background: linear-gradient(180deg, #258AE7 0%, rgba(175, 204, 232, 0.95) 100%);
-webkit-background-clip: text;
-webkit-text-fill-color: transparent;
background-clip: text;
text-fill-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="since">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>660</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font-family: 'Georgia';
font-style: normal;
font-weight: 700;
font-size: 29.8444px;
line-height: 34px;
/* identical to box height */

display: flex;
align-items: center;
text-align: center;
color: #002D62 ; 
background: linear-gradient(180deg, #258AE7 0%, rgba(175, 204, 232, 0.95) 100%);
-webkit-background-clip: text;
-webkit-text-fill-color: transparent;
background-clip: text;
text-fill-color: transparent;
</string>
    </property>
    <property name="text">
     <string>since</string>
    </property>
   </widget>
   <widget class="QLabel" name="date_label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>660</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font-family: 'Georgia';
font-style: normal;
font-weight: 700;
font-size: 29.8444px;
line-height: 34px;
/* identical to box height */

display: flex;
align-items: center;
text-align: center;
color: #002D62 ; 
background: linear-gradient(180deg, #258AE7 0%, rgba(175, 204, 232, 0.95) 100%);
-webkit-background-clip: text;
-webkit-text-fill-color: transparent;
background-clip: text;
text-fill-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="phone_number">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>660</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font-family: 'Georgia';
font-style: normal;
font-weight: 700;
font-size: 29.8444px;
line-height: 34px;
/* identical to box height */

display: flex;
align-items: center;
text-align: center;
color: #002D62 ; 
background: linear-gradient(180deg, #258AE7 0%, rgba(175, 204, 232, 0.95) 100%);
-webkit-background-clip: text;
-webkit-text-fill-color: transparent;
background-clip: text;
text-fill-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="change_password">
    <property name="geometry">
     <rect>
      <x>1002</x>
      <y>30</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="oldPassword">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>90</y>
      <width>261</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>old Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_password">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>130</y>
      <width>261</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>New Password</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
